Release `3.1.0` to `main` (#503)
stellar/stellar-disbursement-platform-backend: CHANGELOG.md

> [!WARNING]  
> This version is only compatible with the [stellar/stellar-disbursement-platform-frontend](https://github.com/stellar/stellar-disbursement-platform-frontend) version `3.1.0`.

### Added

- Export functionality, allowing users to export:
  - Disbursements with filters. [#490](https://github.com/stellar/stellar-disbursement-platform-backend/pull/490)
  - Payments with filters. [#493](https://github.com/stellar/stellar-disbursement-platform-backend/pull/493)
  - Receivers with filters. [#496](https://github.com/stellar/stellar-disbursement-platform-backend/pull/496)
- Option to delete a disbursement in `DRAFT` or `READY` status. [#487](https://github.com/stellar/stellar-disbursement-platform-backend/pull/487)
- Add futurenet as one of the e2e tests scenarios applied to the e2e GitHub Action. [#472](https://github.com/stellar/stellar-disbursement-platform-backend/pull/472)

### Changed

- Update Circle API to use Circle payouts, which is the new officially suggested (and supported) API. [#486](https://github.com/stellar/stellar-disbursement-platform-backend/pull/486), [#491](https://github.com/stellar/stellar-disbursement-platform-backend/pull/491)

> - Only execute the GiyHub e2e tests workflow prior to publishing Docker images, removing it from the pull requests test suite. [#479](https://github.com/stellar/stellar-disbursement-platform-backend/pull/479)

Reviewer: ```suggestion
- Only execute the GitHub e2e tests workflow prior to publishing Docker images, removing it from the pull requests test suite. [#479](https://github.com/stellar/stellar-disbursement-platform-backend/pull/479)
```

- Simplify docker compose by making Kafka optional and defaulting to scheduled jobs. [#481](https://github.com/stellar/stellar-disbursement-platform-backend/pull/481)
- Make Dashboard User E-mails case insensitive. [#485](https://github.com/stellar/stellar-disbursement-platform-backend/pull/485)

### Fixed

- Fix XLM support on the integration tests. [#470](https://github.com/stellar/stellar-disbursement-platform-backend/pull/470)

> - Fix `main.sh` script so it kills all containers including non-sdp containers. [#480](https://github.com/stellar/stellar-disbursement-platform-backend/pull/480)

Reviewer: ```suggestion
- Fix `main.sh` script so that it doesn't kill non-sdp containers. [#480](https://github.com/stellar/stellar-disbursement-platform-backend/pull/480)
```
